enhancements/baremetal/coreos-image-in-release.md

```diff
@@ -0,0 +1,269 @@
+---
+title: coreos-image-in-release
+authors:
+  - "@zaneb"
+reviewers:
+  - "@hardys"
+  - "@dtantsur"
+  - "@elfosardo"
+  - "@sadasu"
+  - "@kirankt"
+  - "@asalkeld"
+  - "@cgwalters"
+  - "@aravindhp"
+  - "@jlebon"
+  - "@sosiouxme"
+  - "@dhellmann"
+  - "@sdodson"
+  - "@LorbusChris"
+approvers:
+  - "@hardys"
+  - "@aravindhp"
+creation-date: 2021-09-22
+last-updated: 2021-09-22
+status: implementable
+see-also:
+  - "/enhancements/coreos-bootimages.md"
+---
+
+# Include the CoreOS image in the release for baremetal
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Operational readiness criteria is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+The baremetal platform is switching from the OpenStack QCOW2 CoreOS image to
```

Comment:
@zaneb this enhancement will also assist AI. Maybe we should address both cases here?